packages/sdk/identity/src/offchain-data-wrapper.test.ts

```diff
@@ -142,6 +143,50 @@ testWithGanache('Offchain Data', (web3) => {
     })
   })
 
+  describe('with the DEK being the signer', () => {
+    it('can write a name', async () => {
+      const writerPrivateKey = ACCOUNT_PRIVATE_KEYS[5]
+      const writerDEK = randomBytes(32).toString('hex')
+      const compressedWriter = await setupAccount(writerPrivateKey, writerDEK, true)
+      const DEKAddress = privateKeyToAddress(writerDEK)
+      compressedWriter.wrapper.kit.connection.addAccount(writerDEK)
+      compressedWriter.wrapper.signer = DEKAddress
+
+      const nameAccessor = new PublicNameAccessor(compressedWriter.wrapper)
+      await nameAccessor.write(testPayload)
+
+      const readerNameAccessor = new PublicNameAccessor(reader.wrapper)
+      const resp = await readerNameAccessor.readAsResult(compressedWriter.address)
+      if (resp.ok) {
```

Comment:
nitpick: This is repeated in multiple test cases so we can create a common method